server/authentication/auth.ts

```diff
@@ -14,6 +17,24 @@ passport.deserializeUser((user, done) => {
   done(null, user as Express.User)
 })
 
+export type AuthenticationMiddleware = () => RequestHandler
+
+const authenticationMiddleware: AuthenticationMiddleware = () => {
```

Comment:
copied same pattern as in typescript template

Reply:
Ah this might mean the return url might actually start working !